sklearn/metrics/ranking.py

```diff
@@ -250,7 +254,8 @@ def _binary_roc_auc_score(y_true, y_score, sample_weight=None):
             raise ValueError("Only one class present in y_true. ROC AUC score "
                              "is not defined in that case.")
 
-        fpr, tpr, tresholds = roc_curve(y_true, y_score,
+        # use closure for pos_label parameter
```

Comment:
It would be better to make it explicit and not use closure. Just add the `pos_label` arg to `_binary_roc_auc_score` and pass it explicitly...